packages/graphql/src/schema/validation/custom-rules/directives/relationship.ts

```diff
@@ -176,5 +180,18 @@ function validateRelationshipTarget(
                 []
             );
         }
+    } else if (relatedType.kind === Kind.INTERFACE_TYPE_DEFINITION) {
```

Comment:
Do you want to add the union target validation here or in a follow-up PR?

Reply:
Yes, let's keep this one only to interfaces